Core/Main/elxParameterObject.cxx

```diff
@@ -167,6 +167,39 @@ ParameterObject::GetParameter(const unsigned int index, const ParameterKeyType &
 }
 
 
+/**
+ * ********************* HasParameter *********************
+ */
+
+bool
+ParameterObject::HasParameter(const unsigned int index, const ParameterKeyType & key) const
+{
+  if (index >= m_ParameterMaps.size())
+  {
+    return false;
+  }
+  return m_ParameterMaps[index].find(key) != m_ParameterMaps[index].end();
```

Comment:
Mostly looks fine to me, thanks Matt! In this particular case, I would rather avoid calling `m_ParameterMaps[index]` twice, by doing, instead:

```cpp
    return m_ParameterMaps[index].count(key) > 0;
```
Of course, the ideal would be `m_ParameterMaps[index].contains(key)`, but that's for C++20  😸

Reply:
Good idea, Neils! Added.